src/KeyDown.tsx

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
+import React, { KeyboardEvent } from 'react';
 
-interface KeyDownProps {
+export interface KeyDownProps {
   children: JSX.Element | string;
   up: () => void;
   down: () => void;
   left: () => void;
   right: () => void;
   enter: () => void;
+  onKeyDown?: (event: KeyboardEvent<HTMLDivElement>) => any;
```

Comment:
Since this function doesn't return any value, should it be `(event: KeyboardEvent<HTMLDivElement>) => void`?